### DISCUSSION TITLE
Error in documentation example of the push_to_git_remote action?

### DISCUSSION THREAD

**Discussion by jey-avono** [fastlane/fastlane#22222] [General]
The first example in the documentation of the **push_to_git_remote** action under https://docs.fastlane.tools/actions/push_to_git_remote/ says

`push_to_git_remote # simple version. pushes "master" branch to "origin" remote`

My reading of the rest of the documentation and the code suggests that this would not push the _"master" branch_, but the _current branch_. This seems to me a remnant of previous behaviour.

Am I missing something ~or should I open an issue for this~? **Edit:** I opened #22226

**Comment by james589goff** [comment]
Hello,
You're spot on! The current behavior of push_to_git_remote pushes the current branch rather than the "master" branch. This might indeed be a remnant of previous versions. Opening an issue, like you did with #22226, is a great step to address and potentially rectify the documentation. [NC Ed Cloud](https://www.ncedcloudlogin.com)

Thanks for taking the initiative to contribute to the community! Does this cover what you needed?

Best Regards,
James Goff
